pkgs/development/python-modules/pyqt-builder/0005-nix-use-NIX_BUILD_CORES-to-build-faster-by-default.patch

```diff
@@ -0,0 +1,28 @@
+From 0a13388f7edc998fb502cfd8a44e4793577b7ae2 Mon Sep 17 00:00:00 2001
+From: Milan Hauth <milahu@gmail.com>
+Date: Mon, 13 Jun 2022 14:43:09 +0200
+Subject: [PATCH 5/5] nix: use NIX_BUILD_CORES to build faster by default
+
+---
+ pyqtbuild/builder.py | 3 +++
+ 1 file changed, 3 insertions(+)
+
+diff --git a/pyqtbuild/builder.py b/pyqtbuild/builder.py
+index 1649186..32349a1 100644
+--- a/pyqtbuild/builder.py
++++ b/pyqtbuild/builder.py
+@@ -293,8 +293,11 @@ class QmakeBuilder(Builder):
+         options = super().get_options()
+ 
+         # Add our new options.
++
++        default_jobs = os.environ.get("NIX_BUILD_CORES", 1)
+         options.append(
+                 Option('jobs', option_type=int,
++                        default=default_jobs,
```

Comment:
should we make this conditional on whether `enableParallelBuilding` is set?

Reply:
yes

https://github.com/NixOS/nixpkgs/blob/9585acba1c3d8c3bcbd53f0a48d884d3070659aa/pkgs/stdenv/generic/setup.sh#L1078